.github/workflows/deploy-standalone-plugins.yml

```
@@ -8,6 +8,9 @@ on:
       slug:
         type: string
         description: 'The slug of the plugin to deploy'
+      dry-run:
+        type: boolean
+        description: 'True to set debug mode'
```

This text will show up next to the checkbox in the workflow UI, perhaps we can make it clearer what it does? We can also add a default here to make it more clear?

```suggestion
        description: 'Debug mode (run without publishing).'
        default: false
```